app.js

```diff
@@ -22,4 +22,14 @@ app.get('/', (req, res) => {
   res.render('index');
 });
 
-app.listen(3000, () => console.log('🏃‍ on port 3000'));
+app.get('/beers', (req, res) => {
+  punkAPI.getBeers().then(beers => res.render('beers', {beerList: beers}))
+  .catch(error => console.log(error)); 
+});
+
+app.get('/random-beer', (req, res) => {
+  punkAPI.getRandom().then(beer => res.render('random-beer', {randomBeer: beer}))
```

Comment:
1. I would distribute this code over a few lines
2. You could send the single random beer, by retrieving it from the array, instead of sending an array to the front end.